About.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Black Girl Magic</title>
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+   <header>
+     <ul>
+        <div>
+            <li><a href="home.html">HOME</a></li>
+        </div>
+        <div>
+            <li><a href="About.html">ABOUT</a></li>
+        </div>
+        <div>
+            <li><a href="Contact.html">CONTACT</a></li>
+        </div>
+        <div>
+            <li><a href="profile.html">PROFILE</a></li>
+        </div>
+        <div>
+            <li><a href="notification.html">NOTIFICATION</a></li>
+        </div>
+     </ul>
+   </header> 
+
+   <section>
+    <div class="container">
+        <h1>ABOUT US</h1>
+        <br>
```

Comment:
Instead of using this you can add margin bottom on your h1 to get the desired space